docs/UserGuide.adoc

```diff
@@ -83,150 +96,368 @@ This symbol indicates that there is something you should be careful to avoid.
 
 === Command Format
 
-All the commands written in the user guide follow some rules that have been mentioned
-below. This will help you better understand how to use the commands.
+This section shows you how all the commands in this guide have been formatted.
 
-[width="%",cols="<20%,<40,options="header",]
+Here is a table outlining how the commands have been formatted in this guide:
```

Comment:
doesn't putting both sentences  make it repetitive? Especially since they're back to back.